src/definition.ts

```diff
@@ -16,8 +16,9 @@ export interface IPoll {
     totalVotes: number;
     votes: Array<IVoter>;
     finished?: boolean;
-    confidential?: boolean;
+    visibility?: string;
```

Comment:
I believe this properly has some finite values like `mixed`, etc right? So y not make an enum for it?